local-storage.js

```diff
@@ -0,0 +1,34 @@
+// Local Storage
+// Declaring Variables to get form inputs
+const storageBook = document.querySelector('#book');
+const storageAuthor = document.querySelector('#author');
+
+// If data is eneterd previously and saved in local storage, retrieve it
+const localSavedData = JSON.parse(localStorage.getItem('formSaved'));
+// When the user loads the page, if there is any data in the local storage
+// the input fields are pre-filled with this data.
+if (localSavedData) {
+  // Declaring variable and assinging values of object keys
+  const localBook = localSavedData.title;
+  const localAuthor = localSavedData.author;
+
+  // assigning keys the values from form inputs
+  storageBook.value = localBook;
+  storageAuthor.value = localAuthor;
+}
+// function to preserve data in browser
```

Comment:
Good job by using local storage to store your data but this is not the correct way as your code is updating a single object  with the new inputs and the idea is to store all added items in local storage for persistency so that even when closing your app the data remains availlable for further use , to solve this call localStorage.setItem after adding or deleting items from your books list in scripts.js also call localStorage.getItem on page load to get whats in localStorage